docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,177 +575,198 @@ The following features are not supported:
-| `Slices`            | `integer`           | The number of slices going around the shape. |
-| `Stacks`            | `integer`           | The number of layers going down the shape. |
-| `ScaleTopRadius`    | `float`             | For conical frustums, the top circle scale radius. |
-| `ScaleBottomRadius` | `float`             | For conical frustums, the bottom circle scale radius. |
-| `ScaleHeight`       | `float`             | For conics, the scale height.    |
-| `ScaleRadius`       | `float`             | For cylinders, the scale radius.  |
-| `ScaleDimensions`   | `Vector3`           | For cuboids, the dimensions. They scale in the same way as for a 9-patch image. |
-| `BevelPercentage`   | `float`             |  For cuboids, the amount of beveling is based on the cuboid's smallest dimension.|
-| `BevelSmoothness`   | `float`             |  For cuboids, the beveled edge smoothness. |
-| `LightPosition`     | `Vector3`           | The position, in the stage space, of the point light, that apply lighting to the model. |
+**Table: MeshVisual properties**
+
+| Property         | Type                | Required              | Description                              |
+|------------------|---------------------|-----------------------|------------------------------------------|
+| `ObjectURL`      | `string`            | Yes                   | The location of the `.obj` file.         |
+| `MaterialtURL`   | `string`            | No                    | The location of the `.mtl` file. Leave blank for a textureless object. |
+| `TexturesPath`   | `string`            | Yes if using materials| The path to the directory where the textures (including gloss and normal) are stored. |
+| `ShadingMode`    | `MeshVisualShadingModeValue` | No           | The type of the shading mode that the mesh uses (see tab. [with possible values](#table_mesh_possible_values)). |
+| `UseMipmapping`  | `bool`              | No                    | Specifies whether to use mipmaps for textures (by default `true`). |
+| `UseSoftNormals` | `bool`              | No                    | Specifies whether to average normals at each point to smoother the textures (by default is `true`). |
+| `LightPosition`  | `Vector3`           | No                    | The position, in the stage space, of the point light that applies lighting to the model (all zeros indicate the top-left corner). |
+
+-   The `ShadingMode` property defines the shading mode type used by the mesh.
 
--   You can select from seven `Shape` values, some of which are specializations as compared to another.
+<a name="table_mesh_possible_values"></a>
+**Table: Shading mode values**
 
-    **Table: Shape values**
+| Enumeration value                      | Description                              |
+|----------------------------------------|------------------------------------------|
+| `TexturelessWithDiffuseLighting`       | Simplest, one color that is lit by ambient and diffuse lighting. |
```

Comment:
| `TexturelessWithDiffuseLighting`       | Simplest, one color that is lit by ambient and diffuse lighting. |

Doubt: What does simplest represent?

Reply:
the simplest method
I removed this word:
Uses one color that is lit by ambient and diffuse lighting.